### PR TITLE
Display fiat equivalents when sending coins

### PR DESCRIPTION
**Pull request by SylEze**
Solves #504 

Old behavior:
send_coins_without_fiat_eqivalent

New behavior:
send_coins_with_fiat_equivalent

### CODE REVIEW

File: src/atomicdex/pages/qt.wallet.page.cpp [hunk 1]
```diff
@@ -270,6 +287,19 @@ namespace atomic_dex
                 {
                     j_out["withdraw_answer"]["fee_details"]["amount"] = j_out["withdraw_answer"]["fee_details"]["miner_fee"];
                 }
+                
+                // Add fees amount in fiat currency.
+                auto fee = j_out["withdraw_answer"]["fee_details"]["amount"];
```
Comment (Milerius) [inline src/atomicdex/pages/qt.wallet.page.cpp:292]: Spurious automatic conversion to `std::string` will rather prefer to use `get<std::string>()` to be sure.

**Comment by naezith** [comment]
Seems good to me!

**Review by tonymorony — CHANGES_REQUESTED** [review]
looks like USD prices calculating not correct in some cases (was my first test to broadcast KMD)
unfortunately atm 1 KMD cost a bit less than 41$ :)

image
image

**Comment by SylEze** [comment]
> looks like USD prices calculating not correct in some cases (was my first test to broadcast KMD)
> unfortunately atm 1 KMD cost a bit less than 41$ :)
> 
> image
> image

I will check the function which does the conversion in the backend and the one which formats the result in the front-end

**Comment by SylEze** [comment]
@tonymorony Can you try again with the last pull and tell me if it's working now please ?

**Review by tonymorony — APPROVED** [review]
lgtm now, thanks!

**Pull request merged**